autoreduce_frontend/reduction_viewer/filters.py

```diff
@@ -0,0 +1,93 @@
+from autoreduce_db.reduction_viewer.models import Experiment, ReductionRun
+from django_filters.filters import CharFilter, Filter
+from django_filters.widgets import RangeWidget
+from django_filters import FilterSet, DateFromToRangeFilter
+from django.db.models import Q
+import re
+from django.core.exceptions import ValidationError
+
+
+class ReductionRunFilter(FilterSet):
+    def validate_run_number(value):
+        if "," in value and "-" not in value:
+            if not re.match('\d+,\d+', value):
+                raise ValidationError("There must be a run number before and after the comma.")
+        elif "-" in value and "," not in value:
+            if not re.match('\d+-\d+', value):
+                raise ValidationError("There must be a run number before and after the hyphen.")
+        elif "-" and "," in value:
+            if not re.match('\d+-\d+', value):
+                raise ValidationError("There must be a run number before and after the hyphen.")
+        else:
+            if not re.match('\d', value):
```

Comment:
```suggestion
            if not re.match('^\d+$', value):
```
Change to match __only__ a number, otherwise input like `6312"£!"£!` causes an error too!